### PR TITLE
Feature | Parent & Subs | Staff Records Navigation

### PR DESCRIPTION
**Pull request by ChazUK**
There are a few more fixes in here which were around the API using the establishment ID stored in the Service

### CODE REVIEW

**Comment by ChazUK** [comment]
[(3) Parent and Subsidiaries - Add Workers](https://trello.com/c/oGW3inOf/53-3-parent-and-subsidiaries-add-workers)

File: src/app/features/workers/basic-records-save-success/basic-records-save-success.component.ts [hunk 1]
```diff
@@ -9,13 +9,15 @@ import { WorkerService } from '@core/services/worker.service';
...
 
-  constructor(private router: Router, private workerService: WorkerService) {}
+  constructor(private route: ActivatedRoute, private router: Router, private workerService: WorkerService) {}
 
   ngOnInit() {
     this.total = this.workerService.getCreateStaffResponse();
+
     if (this.total === 0) {
-      this.router.navigate(['/dashboard'], { fragment: 'staff-records', replaceUrl: true });
+      this.router.navigate(this.return.url, { fragment: 'staff-records', replaceUrl: true });
```
Comment (omarCreativeDev) [inline src/app/features/workers/basic-records-save-success/basic-records-save-success.component.ts]: where and when is the return url being set? doesn't seem to be set by default on line 12

File: src/app/features/workers/staff-details/staff-details.component.ts [hunk 2]
```diff
@@ -56,6 +56,7 @@ export class StaffDetailsComponent extends QuestionComponent implements OnInit,
   }
 
   renderInEditMode() {
+    console.log('edit mode');
```
Comment (omarCreativeDev) [inline src/app/features/workers/staff-details/staff-details.component.ts]: console log

File: src/app/features/workers/basic-records-save-success/basic-records-save-success.component.ts (hunk 1 above)
Comment (ChazUK) [reply src/app/features/workers/basic-records-save-success/basic-records-save-success.component.ts]: Good spot, updated

File: src/app/features/workers/staff-details/staff-details.component.ts (hunk 2 above)
Comment (ChazUK) [reply src/app/features/workers/staff-details/staff-details.component.ts]: removed

**Pull request merged**